.github/workflows/e2e-component-tests-embedding-sdk.yml

```diff
@@ -95,13 +97,52 @@ jobs:
         with:
           name: metabase-ee-${{ github.event.pull_request.head.sha || github.sha }}-uberjar
 
+  build-embedding-sdk:
+    needs: [files-changed, get-build-requirements]
+    if: |
+      !cancelled() &&
+      needs.files-changed.outputs.e2e_all == 'true'
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    env:
+      MB_EDITION: ee
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+
+      - name: Prepare back-end environment
+        uses: ./.github/actions/prepare-backend
+        with:
+          m2-cache-key: "cljs"
+
+      - name: Compile CLJS
+        run: yarn build-pure:cljs
+        shell: bash
+
+      - name: Build Embedding SDK package
+        run: yarn build-embedding-sdk
+
+      - name: Upload Embedding SDK artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: embedding-sdk-build
+          path: resources/embedding-sdk
+
   e2e-tests:
-    needs: [get-build-requirements, build]
+    needs: [get-build-requirements, build-uberjar, build-embedding-sdk]
     if: |
-      !cancelled() && needs.build.result == 'success'
+      !cancelled() &&
+      needs.build-uberjar.result == 'success' &&
+      needs.build-embedding-sdk.result == 'success'
     runs-on: ubuntu-22.04
     timeout-minutes: 45
-    name: e2e-tests-embedding-sdk
+    strategy:
+      matrix:
+        react-version: [18, 19]
```

Comment:
17 is incompatible currently with master, see [this Slack thread](https://metaboat.slack.com/archives/C063Q3F1HPF/p1742452028403089). I'll add it to 52 and 53 branch though.